src/css/AdminStatisticsCss.css

```diff
@@ -138,7 +138,7 @@
     background: #0c1112;
     height: 96%;
     /*min-height: 40vh;*/
-    box-shadow: 1px 1px 10px #3399cc;
+    box-shadow: 1px 1px 10px #8e44ad;
```

Comment:
This variable appears multiple times. As now we can use less css, you can directly change this file to `AdminStatisticsCss.css` to `.less` and use less variables.